syntax = "proto3";


service AuthService {
rpc ValidateToken (TokenRequest) returns (UserResponse);
}


message TokenRequest {
string token = 1;
}


message UserResponse {
int32 user_id = 1;
string email = 2;
bool is_active = 3;
}